src/gui/views/vault_view.py

```
@@ -20,81 +23,322 @@ def __init__(self, api_client: APIClient, user_session: UserSession, parent=None
+            layout = QVBoxLayout(self)
+            error_label = QLabel(f"Error during retry: {str(e)}")
+            error_label.setWordWrap(True)
+            error_label.setStyleSheet("color: red;")
+            layout.addWidget(error_label)
+            
+            # Add retry button
+            retry_btn = QPushButton("Retry")
+            retry_btn.clicked.connect(self.retry_initialization)
+            layout.addWidget(retry_btn)
+            
+        except Exception as e:
```

Ruff (0.8.2)

try-except block with duplicate exception `Exception`

(B025)